data/content/airspace/region/Canada_Airspace.txt

```diff
@@ -5494,15 +5494,15 @@ DP 46:27:10.2N 84:28:32.5W
 DP 46:27:38.7N 84:33:24.6W
 DP 46:38:03.4N 84:45:46.4W
 *END OF BORDER
-V D=+
+V D=+ 
```

Comment:
Can you remove the trailing whitespace on all instances of V D=+ please?
That makes the diff huge, and there is no actual change here.

Reply:
It will take a very long time to manually delete each of those spaces.  I am not sure why they are there, it is the same code used from the previous version.  

The differences are in the airways to the bottom.  Some are removed, some have slightly different routing.